### PR TITLE
Warsaw circle

### PR DESCRIPTION
**Pull request by JoeF131**
Split previous pull request #913 into two.

### CODE REVIEW

**Comment by yhx-12243** [comment]
https://github.com/pi-base/data/pull/913#issuecomment-2477720818

As mentioned earlier, can you simply add the proof that Warsaw circle is simply connected?

**Review by yhx-12243 — CHANGES_REQUESTED** [review]
```suggestion
---
space: S000205
property: P000200
value: true
refs:
- mathse: 3705688
  name: To show that Warsaw circle is simply connected
---

Asserted in {{mathse:3705688}}.
```

**Comment by prabau** [comment]
Comment from @JoeF131 (repeated from #913 for convenience):

> Aren't there more than once space called the "Warsaw circle"? If you find more than one, can you list a few here in the comments so we can discuss which one would be the best one to pick. Or maybe I am confused about that and there is essentially only one.

There are many definitions I found which are all homeomorphic.
+Wikipedia just draws a picture.
+[This paper](https://math.ucr.edu/~res/math205B-2012/polishcircleA.pdf) which was in the reference section on Wikipedia defines it the way I did.
+{{mathse:1004837}} defines it as (the closed sine curve)[https://topology.pi-base.org/spaces/S000114] + an arc
There are some others which we're also some variation of these, but I'm open to change the definition for some other when suggested

----
Comment from @prabau (repeated from #913):

A few other variants: At least it seems one always wants the space to be compact.

https://ncatlab.org/nlab/show/Warsaw+circle.  Not sure if that's the same as in Borsuk's "Theory of Shape" book.
(See also https://math.stackexchange.com/a/1480692)

https://mathoverflow.net/questions/280806/ has two pictures.  The one on the left is not homeomorphic to the other ones, I think.

Note: I don't necessarily disagree with your choice, but wanted to put the other variants out there for discussion if needed.

**Comment by prabau** [comment]
I checked various papers in the literature that use the "Warsaw circle" and it seems the variant you have it the one commonly used, which is the one with the picture in Wikipedia, as well the paper you mentioned, and Munkres.
For example:
https://www.sciencedirect.com/science/article/pii/S0166864113002435
(definition in section 3, with nice picture)

https://www.sciencedirect.com/science/article/pii/S0166864101000189
(description on p. 184)

So we can go with that one.

One thing though.  I know it makes no difference, but most definitions choose the additional arc connecting to right endpoint of the $\sin(1/x)$ graph to the vertical segment on the left by going under the function graph instead of above.  Same thing in various mathse posts, and Wikipedia, and the "paper" you mentioned.  Would it be possible to change your definition to do the same?

**Comment by yhx-12243** [comment]
> I checked various papers in the literature that use the "Warsaw circle" and it seems the variant you have it the one commonly used, which is the one with the picture in Wikipedia, as well the paper you mentioned. For example: https://www.sciencedirect.com/science/article/pii/S0166864113002435 (definition in section 3, with nice picture)
> 
> https://www.sciencedirect.com/science/article/pii/S0166864101000189 (description on p. 184)
> 
> So we can go with that one.
> 
> One thing though. I know it makes no difference, but most definitions choose the additional arc connecting to right endpoint of the $sin⁡ ( 1 / x )$ graph to the vertical segment on the left by going under the function graph instead of above. Same thing in various mathse posts, and Wikipedia, and the "paper" you mentioned. Would it be possible to change your definition to do the same?

So as you mentioned, in fact we have three different type of "Warsaw circles" that *are not homeomorphic each other*, where I'm tend to the first one (just the original one, like Wikipedia):
war

**Comment by prabau** [comment]
S205 README: You can compare with most of the other spaces.  In the description of the spaces we usually don't repeat the name of the space in the text itself.  That allows to easily change just the title later on if needed (and the new name will propagate everywhere via the `{S205}` syntax).

**Comment by StevenClontz** [comment]
My two cents: I've always thought of the first illustration in https://github.com/pi-base/data/pull/915#issuecomment-2480365160 when folks talk about this example. My expectation is that, as far as priorities we track in pi-Base, the choice is going to be arbitrary, so we shouldn't overthink the choice. We can always add others if they are needed later (and call them all "Warsaw circles" - that's why we have numerical IDs and aliases).

**Comment by StevenClontz** [comment]
(This thread is making me think that we will benefit from support for some kind of illustrative markup within our Markdown...)

**Comment by prabau** [comment]
> My two cents: I've always thought of the first illustration in [#915 (comment)](https://github.com/pi-base/data/pull/915#issuecomment-2480365160) when folks talk about this example. My expectation is that, as far as priorities we track in pi-Base, the choice is going to be arbitrary, so we shouldn't overthink the choice. We can always add others if they are needed later (and call them all "Warsaw circles" - that's why we have numerical IDs and aliases).

Note: FYI, the third example above is what Hatcher calls "quasi-circle". (He does not use the Warsaw circle name.)

**Comment by yhx-12243** [comment]
> > My two cents: I've always thought of the first illustration in [#915 (comment)](https://github.com/pi-base/data/pull/915#issuecomment-2480365160) when folks talk about this example. My expectation is that, as far as priorities we track in pi-Base, the choice is going to be arbitrary, so we shouldn't overthink the choice. We can always add others if they are needed later (and call them all "Warsaw circles" - that's why we have numerical IDs and aliases).
> 
> Note: FYI, the third example above is what Hatcher calls "quasi-circle". (He does not use the Warsaw circle name.)

In fact there are [some places](https://wildtopology.com/bestiary/warsaw-circle/) which call it Warsaw circle.

But it doesn't matter, we'll still tend to use the first.

**Comment by JoeF131** [comment]
> S205 README: You can compare with most of the other spaces. In the description of the spaces we usually don't repeat the name of the space in the text itself. That allows to easily change just the title later on if needed (and the new name will propagate everywhere via the `{S205}` syntax).

I wanted to keep the description similar to the description of [P113](https://topology.pi-base.org/spaces/S000113),[P114](https://topology.pi-base.org/spaces/S000114),[P115](https://topology.pi-base.org/spaces/S000115) since they are all related. Should I change there description in this PR as well or should I create a new one just for these?

For the Warsaw circle one might add that there are some non homeomorphic spaces with the same name. Here it would be nice to have a link to the image in https://github.com/pi-base/data/pull/915#issuecomment-2480365160 so that I could refer to it.

File: spaces/S000205/README.md
Comment (prabau) [inline spaces/S000205/README.md:8]:
```suggestion
refs:
  - wikipedia: Shape_theory_(mathematics)#Warsaw_circle
    name: Warsaw circle on Wikipedia
  - doi: 10.1016/j.topol.2013.07.001
    name: Milnor–Thurston homology groups of the Warsaw Circle (J. Przewocki)
---

The set $X=\{(x, \sin\frac{1}{x})\mid x \in (0,1]\} \cup \{(0,y)\mid y \in [-2,1]\} \cup \{(x,-2)\mid x\in[0,1]\} \cup \{(1,x)\mid x \in [-2,\sin 1]\}$ with the subspace topology inherited from $\mathbb{R}^2$.

It consists of {S114} together with an extra arc joining its rightmost point to the point $(0,-2)$.
See {{wikipedia:}} or section 3 of {{doi:10.1016/j.topol.2013.07.001}}

Note: There are a few closely related but non-isomorphic variants of the *Warsaw circle*.
The description chosen here seems to be the most common.
```

**Comment by prabau** [comment]
Added a suggestion for the README.  

Our references should preferably point to an authoritative source, not a comment of ours in github.  So added a reference to a paper using the Warsaw circle and pointed to it and wikipedia for an illustration.  No need to refer to the other variants here.

Also moved the extra arc to be below the $\sin(1/x)$ graph, as that's more common.

**Comment by prabau** [comment]
> I wanted to keep the description similar to the description of [P113](https://topology.pi-base.org/spaces/S000113),[P114](https://topology.pi-base.org/spaces/S000114),[P115](https://topology.pi-base.org/spaces/S000115) since they are all related. Should I change there description in this PR as well or should I create a new one just for these?

(I think you meant S113, etc)  Yeah, if you want, you can do it in the same PR, as that should be a trivial change.

**Comment by StevenClontz** [comment]
> Our references should preferably point to an authoritative source, not a comment of ours in github.

+1 I've pushed for the scope of pi-Base to focus on reflecting peer-reviewed mathematics, not peer-reviewing mathematics. So if something needs a reference, we should point to the literature or at least more robust forums such as MathOverflow or Mathematics StackExchange.

See [the relevant bit of the wiki](https://github.com/pi-base/data/wiki/Reviewing#:~:text=If%20the%20result%20is%20not%20directly%20addressed%20by%20a%20source%2C) (which can be revisited, but that would require community discussion).

**Comment by prabau** [comment]
P200 and P199: The referenced justifications do more that just "assert" some facts.  Better say `See {{mathse:...}}.`.

**Comment by JoeF131** [comment]
So I have now changed these last two things.

I didn't actually mean to refer to the comment in github but rather to the source of the image. (unless if it's selfmade) 
Now I think, that this isn't necessary anymore with the way the README is formulated.

**Comment by prabau** [comment]
P199: Whenever we use a `{{mathse:...}}` link, we should also put the title of the post in the `refs:` section in the metadata portion of the file.

Also, the whole post in this case is about the topic we want.  So we can refer to the whole question, i.e., using mathse id 1004837 instead of the particular answer with id 1004886.

**Comment by prabau** [comment]
FYI, I was not able to load the latest build from the pi-base web page.  Maybe because I had an incorrect `{{wikipedia:}}`.  Now fixed, linking directly to "Warsaw circle", which is a wikipedia redirect.

You also completely changed the meaning of S114 (Closed topologist's sine curve).  I fixed it back.

**Comment by prabau** [comment]
Also, not sure why you insist on using mathse 1004886 instead of the whole post.  I'll fix it.

**Comment by prabau** [comment]
OK. Now things are usable again when looking at this branch on the pi-base web site.

Not sure if you have played with it yet, but in https://topology.pi-base.org/spaces/S000205/properties, to right of the filter search box, there are a bunch of icons that can be clicked.  One of them (person icon) lists the asserted properties.  As a general rule, we try to have a minimal set of asserted properties, from which most of the other ones can be derived.

Can you check if some asserted properties are redundant?

**Comment by JoeF131** [comment]
> Also, not sure why you insist on using mathse 1004886 instead of the whole post. I'll fix it.

Well I mistakenly thought the second number references the answer so I replaced mathse:1004886/1469194 by mathse:1004886. My bad.

**Comment by prabau** [comment]
> I've removed [indiscrete](https://topology.pi-base.org/properties/P000129) from all four spaces and [arc connected]([https://topology.pi-base.org/spaces/P000038]](https://topology.pi-base.org/spaces/P000038%5D) from the [Warsaw circle](https://topology.pi-base.org/spaces/S205) since they are redundant.
> 
> There are still some traits for all 4 spaces that are unknown but none seem to be obviously true or false.
> 
> Edit: I initially also removed [path connected](https://topology.pi-base.org/properties/P000037), [biconnected](https://topology.pi-base.org/properties/P000044) from [S114](https://topology.pi-base.org/spaces/S114) but they are necessary so I undid this.

Thanks for cleaning up some of redundant traits.  I had mentioned the redundant traits thinking mainly of the new space (Warsaw circle) that you are adding.  In the future, **please do not enlarge the scope of what you are doing** one more time.  The cleanup of the others could have been done in a separate PR.

**Comment by prabau** [comment]
Finally, to get to the end of this never-ending Warsaw circle PR, one more comment.

P41 (locally connected): Why are you comparing with the Extended topologist's sine curve?
The Closed topologist's sine curve (S114) seems a much more natural choice, as it is subspace of S205 (see the README file) and it contains the portion of interest to show S205 is not locally connected.

Instead of saying `Similar to {S114}.`, we have a better way of doing this by pointing directly to the justification that S114 is not P41.  The syntax is: `Similar to {S114|P41}.`. 
We have many examples of that syntax.  E.g., see https://topology.pi-base.org/spaces/S000005/properties/P000087

**Comment by prabau** [comment]
> Well I mistakenly thought the second number references the answer so I replaced mathse:1004886/1469194 by mathse:1004886.

FYI if you are curious, the second number (1469194) is your own mathse user id.
https://math.stackexchange.com/users/1469194

**Comment by prabau** [comment]
Looks good now.  Approving.

We really appreciate your contribution.  I hope you can review your past two or three PRs with all the comments that were given, so you can see the kind of things to pay attention to for future contributions.  That will make things go more smoothly.
But again, thank you for this.

**Review by prabau — APPROVED** [review]

**Comment by prabau** [comment]
@yhx-12243 As you requested changes, please review the changes and approve if you agree, so we can proceed with the merge.

I can do the squash and merge afterwards.  If you want to do the squash and merge yourself, please be very careful to first blank out the associated description before clicking on "squash and merge".  In this case, it would be a sorry thing to have to see a ridiculous list of 21 inconsequential individual commits in the `git log`.  (as already discussed a few times)

**Review by yhx-12243 — APPROVED** [review]

**Pull request merged**
